### DISCUSSION TITLE
/pixiv 后请求错误了

### DISCUSSION THREAD

**Discussion by cxplay** [y-young/nazurin#12] [Q&A]
![image](https://user-images.githubusercontent.com/62034099/107992315-653b3280-7013-11eb-8f20-317301da061b.png)
哦我的上帝, 这简直要命啊我的老伙计.

**Comment by y-young** [comment]
> ![image](https://user-images.githubusercontent.com/62034099/107992315-653b3280-7013-11eb-8f20-317301da061b.png)
> 哦我的上帝, 这简直要命啊我的老伙计.

请问你是第一次使用pixiv相关命令吗？由于最近Pixiv更改了API认证方式，不再支持使用用户名和密码登录，所以现在需要事先手动获取 `refresh_token`，详见#9
手动获取`refresh_token`的工具请参考 https://gist.github.com/ZipFile/c9ebedb224406f4f11845ab700124362
获取之后请将其填入数据库nazurin集合的pixiv文档中，如下：
```json
{
  "access_token": "",
  "refresh_token": "在此填入",
  "updated_time": 0
}
```
我将尽快更新相关功能，如需要帮助可在此帖回复

**Reply by cxplay** [reply]
啊这, 我应该怎样在使用TinyDB的Heroku里面修改这个参数呢?

**Reply by y-young** [reply]
> 啊这, 我应该怎样在使用TinyDB的Heroku里面修改这个参数呢?

Heroku不建议配合TinyDB使用，因为Heroku Dyno的文件系统是临时的，一段时间后就会清空，所以无法持久地存储数据
建议使用Firebase或者MongoDB
之后将使用环境变量的配置方式，我会尽快更新

**Reply by y-young** [reply]
> 啊这, 我应该怎样在使用TinyDB的Heroku里面修改这个参数呢?

现在1.2版本已经支持使用环境变量配置了
